src/lib/draw/GeometryMode.svelte

```diff
@@ -122,26 +122,45 @@
     });
   }
 
-  // Auto-save
-  for (let tool of [polygonTool]) {
-    tool.addEventListenerUpdated((feature) => {
-      if ($currentMode == thisMode) {
-        gjScheme.update((gj) => {
-          let updateFeature = gj.features.find(
-            (f) => f.id == currentlyEditing
-          )!;
-          updateFeature.geometry = feature.geometry;
-          return gj;
-        });
-      }
-    });
-  }
+  // Auto-save for everything except pointTool
+  polygonTool.addEventListenerUpdated((feature) => {
+    if ($currentMode == thisMode) {
+      gjScheme.update((gj) => {
+        let updateFeature = gj.features.find((f) => f.id == currentlyEditing)!;
+        updateFeature.geometry = feature.geometry;
+        return gj;
+      });
+    }
+  });
+  // TODO Maybe use one callback (here and elsewhere); the type-safety is not really helpful
```

Comment:
I like this idea. Seems like we're gonna be doing the exact same thing in 3+ places which is code smelly

Reply:
A later commit in this PR does this